app/components/provider_interface/user_permission_summary_component.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= tag.dl(class: 'govuk-summary-list') do %>
```

Comment:
I copied the original summary list code from govuk components (we cant use the component because there are some issues with setting it up as part of a `row`) so it made sense to reuse its code so it's easy to compare and bring it up to date if required. 

https://github.com/DFE-Digital/govuk-components/blob/master/app/components/govuk_component/summary_list.html.erb